packages/framework/src/Model/Product/Search/Export/ProductSearchExporter.php

```diff
@@ -53,7 +115,42 @@ public function export(int $domainId, string $locale): void
             $exportedIds = array_merge($exportedIds, $batchExportedIds);
             $startFrom += static::BATCH_SIZE;
         } while (!empty($batchExportedIds));
-        $this->removeNotUpdated((string)$domainId, $exportedIds);
+        $this->removeNotUpdated($domainId, $exportedIds);
+    }
+
+    /**
+     * @param int $domainId
+     * @param string $locale
+     * @param \Symfony\Component\Console\Style\SymfonyStyle $symfonyStyleIo
+     */
+    public function exportWithOutput(int $domainId, string $locale, SymfonyStyle $symfonyStyleIo): void
+    {
+        $this->validateInjectedDependencies();
+
+        $this->sqlLoggerFacade->temporarilyDisableLogging();
```

Comment:
why is the logging disabled? Maybe it would worth to explain it in commit message

Reply:
Force pushed with extended commit message :)